client/src/pages/signin.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Button, TextField, Paper, FormControl } from '@material-ui/core';
+import laptopPhoto from '../images/blue-shirt-at-laptop.png'
+import { Link } from 'react-router-dom';
+import './signin.css';
+
+export default class Signin extends React.Component {
```

Comment:
Try to use functional components with hooks.
[Official React docs](https://reactjs.org/docs/hooks-faq.html#should-i-use-hooks-classes-or-a-mix-of-both) recommend that all new component be written with hooks, as that is the direction React is taking, and class based components will likely stay supported but outdated